### ISSUE TITLE
Media files are missing in generated `vsix` files

### ISSUE THREAD

**Issue by alvsan09** [eclipse-theia/vscode-builtin-extensions#116]
Some media files are missing in generated `vsix` files.
This was found when testing generation of built-ins from `vscode` 1.77.2 and validating the functionality offered by the
`simple-browser` extension.

Testing the `simple-browser` extension from compiled sources under a `vscode` folder it worked as expected, 
however when testing it from a `vsix` file generated by this repository, a `file not found, 404` error is encountered:

From the `chrome` development tools, we see the following related URL:
http://1c72b4ec-ad30-4f3c-8961-cb2a8853cd3b.webview.localhost:3099/webview/theia-resource/file///tmp/vscode-unpacked/simple-browser-1.77.2.vsix/extension/media/index.js

Examining the contents of the `vsix` will show that the `extensions/media/` folder does not contain the expected `index.js` file. 

This issue has been present for a while and seems to impact a small number of extensions i.e.

```
markdown-math
markdown-language-features
ipynb
builtin-notebook-renderers
simple-browser
```

**Comment by marcdumais-work** [comment]
@alvsan09 good catch! 

Since we have not published 1.72.2 yet, I suggest we make a fix on top of my [1.72.2 PR](https://github.com/eclipse-theia/vscode-builtin-extensions/pull/113), and publish it with the fix.

**Comment by marcdumais-work** [comment]
I have investigated quickly, looking into the problem above, with `simple-browser`. 

Indeed, `media/index.js` is missing in our build. I am hoping that your ovsx update PR might help with this - have you checked? 

Else, I can note that `simple-browser` looks to be a bit more difficult to build vs other extensions. It seems that the `scripts` entries in its `package.json` are not successfully called with `yarn`:
```bash
yarn && yarn compile
Error: Cannot find module 'gulp-cli'
Require stack:
- /usr/share/nodejs/gulp/bin/gulp.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:1026:15)
    at Function.Module._load (node:internal/modules/cjs/loader:871:27)
    at Module.require (node:internal/modules/cjs/loader:1098:19)
    at require (node:internal/modules/cjs/helpers:108:18)
    at Object.<anonymous> (/usr/share/nodejs/gulp/bin/gulp.js:3:1)
    at Module._compile (node:internal/modules/cjs/loader:1196:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1250:10)
    at Module.load (node:internal/modules/cjs/loader:1074:32)
    at Function.Module._load (node:internal/modules/cjs/loader:909:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/usr/share/nodejs/gulp/bin/gulp.js' ]
}
error Command failed with exit code 1.
```

However it seems to work if I use `npm` to compile, it works:
```bash
yarn && npm run compile
```

And the `main.js` is present:
```bash
vscode-builtin-extensions/vscode/extensions/simple-browser$ ll media/
total 132
drwxr-xr-x 2 USER eusers   4096 May 19 11:03 ./
drwxr-xr-x 6 USER eusers   4096 May 19 11:01 ../
-rw-r--r-- 1 USER eusers 102647 May 19 11:03 codicon.css
-rw-r--r-- 1 USER eusers   1787 Feb  8 11:03 icon.png
-rw-r--r-- 1 USER eusers   1507 May 19 11:03 index.js
-rw-r--r-- 1 USER eusers   2313 Feb  8 11:03 main.css
-rw-r--r-- 1 USER eusers    312 Feb  8 11:03 preview-dark.svg
-rw-r--r-- 1 USER eusers    312 Feb  8 11:03 preview-light.svg
```

**Comment by alvsan09** [comment]
> @alvsan09 good catch!
> 
> Since we have not published 1.72.2 yet, I suggest we make a fix on top of my [1.72.2 PR](https://github.com/eclipse-theia/vscode-builtin-extensions/pull/113), and publish it with the fix.

We can actually fixed it with the facilities currently available in the current `vscode` sub-module:
see: 
https://github.com/eclipse-theia/vscode-builtin-extensions/pull/117
